pootle/static/js/editor/app.js

```diff
@@ -1341,15 +1341,13 @@ PTL.editor = {
 
   /* reDraws the translate table rows */
   reDraw(newTbody) {
-    const $oldRows = this.$editorBody.find('tr');
-
     // Remove autosize event listeners for textarea before removing
     autosize.destroy(document.querySelectorAll('textarea.expanding'));
 
-    $oldRows.remove();
+    const $oldRows = this.$editorBody.find('tr').remove();
```

Comment:
There is no need to save a `const` since there is no other reference to `$oldRows`.